pages/api/account/holds/update/[id].ts

```diff
@@ -19,16 +19,17 @@ export default async function handler(
     return res.status(responseStatus).json(responseMessage)
   }
 
-  if (req.method !== "POST") {
-    responseMessage = "Please make a POST request to this endpoint."
+  if (req.method !== "PUT") {
```

Comment:
Purely out of curiosity, why did the method need to be changed?

Reply:
we had originally thought that we weren't able to make put requests from the front end, so this was written to listen for posts. but it turns out we can, so it makes sense for the request from the front end to match the one on the backend.